### ISSUE TITLE
Add support for NVME Devices

### ISSUE THREAD

**Issue by mbloch1986** [shinesolutions/packer-aem#207]
Baking with the new instance types which are using NVME devices fails due to a failed inspec test e.g.
```
Profile: tests from dispatcher_spec.rb (tests from dispatcher_spec.rb)
Version: (not specified)
Target:  local://

  Mount /mnt/ebs1
     ✔  is expected to be mounted
     ∅  device is expected to eq "/dev/xvdb"
     
     expected: "/dev/xvdb"
          got: "/dev/nvme1n1"
     
     (compared using ==)

```

**Comment by mbloch1986** [comment]
Since we can't control the decice name with the NVME supported instance type it's probably worth removing the inspec test for it now.

**Comment by mbloch1986** [comment]
Included in release 4.27.0
